guides/source/7_1_release_notes.md

```diff
@@ -65,6 +65,43 @@ TODO: https://github.com/rails/rails/pull/45602
 
 TODO: https://github.com/rails/rails/pull/46049
 
+### Extend `response.parsed_body` to parse HTML with Nokogiri
+
+[Extend the `ActionDispatch::Testing` module][#47144] to support parsing the
+value of an HTML `response.body` into a `Nokogiri::HTML5::Document` instance:
+
+```ruby
+get "/posts"
+
+response.content_type         # => "text/html; charset=utf-8"
+response.parsed_body.class    # => Nokogiri::HTML5::Document
+response.parsed_body.to_html  # => "<!DOCTYPE html>\n<html>\n..."
+```
+
+Newly added [Nokogiri support for pattern matching][nokogiri-pattern-matching],
+along with built-in [Minitest support for pattern
+matching][minitest-pattern-matching] presents opportunities to make test
+assertions about the structure and content of the HTML response:
+
+```ruby
+get "/posts"
+
+html = response.parsed_body # => <html>
+                            #      <head></head>
+                            #        <body>
+                            #          <main><h1>Some main content</h1></main>
+                            #        </body>
+                            #     </html>
+
+assert_pattern { html.at("main") => { content: "Some main content" } }
```

Comment:
These are raising [Rubocop failures with 2.7 syntax](https://github.com/rails/rails/actions/runs/5943003561/job/16117178561#step:4:12). Is there a way to configure Rubocop for this inline snippet? I'd prefer that over wrapping the assertions in `# rubocop:disable Lint/Syntax`, since that could be confusing for readers.

Reply:
Done. Thank you!